gst/nnstreamer/elements/gsttensor_transform.c

```diff
@@ -79,6 +79,12 @@ GST_DEBUG_CATEGORY_STATIC (gst_tensor_transform_debug);
 
 #define REGEX_ARITH_OPTION_TYPECAST "(typecast:([u]?int(8|16|32|64)|float(16|32|64)))"
 
+/**
+ * @brief The transpose rank is fixed to 4.
+ * This RANK does not affect other/tensors(s)'s NNS_TENSOR_RANK_LIMIT.
+ */
+#define NNS_TENSOR_TRANSPOSE_RANK_LIMIT (4)
```

Comment:
If you are using ```->transpose_tank ``` instead of ```G_PARAM_STATIC_STRINGS``` because of typing, add ```U``` to ```4```.

Reply:
I used it because I misunderstood your last comment
Removed `->transpose_rank`!
Thank you :)